files/etc/xrd/bootstrap.sh

```diff
@@ -50,3 +50,7 @@ tuned-adm active
 # Enable and run the hugepage configuration service.
 systemctl start hugetlb-gigantic-pages
 systemctl enable hugetlb-gigantic-pages
+
+# Add boot cmdline args
+non_isolated_cores=$(/etc/xrd/get_non_isolated_cores.sh $ISOLATED_CORES)
+grubby --update-kernel ALL --args "nohz_full=${ISOLATED_CORES} nohz=on skew_tick=1 intel_pstate=disable tsc=reliable nosoftlockup isolcpus=${ISOLATED_CORES} irqaffinity=${non_isolated_cores} hugepages=${boot_hugepages} rcu_nocbs=${ISOLATED_CORES} hugepagesz=1G default_hugepagesz=1G rcupdate.rcu_normal_after_boot=1"
```

Comment:
How have you determined this is the set of required cmdline arguments? Are they following the ones which tuned generates? If so, I think it's worth having a comment here (and probably in README too) explaining that the tuned `bootloader` plugin does not work on EKS, but we match the cmdline args set by its `realtime-virtual-guest` profile (with the additional hugepage settings).

Reply:
This is the set added by TuneD `realtime-virtual-guest` with the additional arguments we previously passed to the bootloader plugin. Adding comment as suggested.